wski18n/resources/en_US.all.json

```diff
@@ -330,5 +330,10 @@
   {
     "id": "path of the .cert file",
     "translation": "path of the .cert file"
+  },
+  {
+    "id": "The name of the application {{.appNameDeploy}} in deployment.yaml does not match the name of the application {{.appNameManifest}}} in manifest.yaml.",
```

Comment:
Although our tests assume the file names are manifest.yaml and deployment.yaml... the user could provide any name for the base file name.

Reply:
Done.